MagneticField/Engine/python/volumeBasedMagneticField_dd4hep_160812_cfi.py

```diff
@@ -0,0 +1,95 @@
+import FWCore.ParameterSet.Config as cms
+
+
+# This cfi contains everything needed to use the VolumeBased magnetic
+# field engine version 160812 built using dd4hep for the geometry.
+#
+# PLEASE DO NOT USE THIS DIRECTLY
+# Always use the standard sequence Configuration.StandardSequences.MagneticField_cff
+
+
+DDDetectorESProducer = cms.ESSource("DDDetectorESProducer",
+    confGeomXMLFiles = cms.FileInPath('MagneticField/GeomBuilder/data/cms-mf-geometry_160812.xml'),
+    rootDDName = cms.string('cmsMagneticField:MAGF'),
+    appendToDataLabel = cms.string('magfield')
+    )
+
+
+DDCompactViewMFESProducer = cms.ESProducer("DDCompactViewMFESProducer",
+                                            appendToDataLabel = cms.string('magfield')
+                                           )
+
+
+ParametrizedMagneticFieldProducer = cms.ESProducer("ParametrizedMagneticFieldProducer",
+    version = cms.string('OAE_1103l_071212'),
+    parameters = cms.PSet(
+        BValue = cms.string('3_8T')
+    ),
+    label = cms.untracked.string('parametrizedField')
+)
+
+
+VolumeBasedMagneticFieldESProducer = cms.ESProducer("DD4hep_VolumeBasedMagneticFieldESProducer",
+    DDDetector = cms.ESInputTag('', 'magfield'),
+    appendToDataLabel = cms.string(''),
+    useParametrizedTrackerField = cms.bool(True),
+    label = cms.untracked.string(''),
+    attribute = cms.string('magfield'),
+    value = cms.string('magfield'),
+    paramLabel = cms.string('parametrizedField'),
```

Comment:
it looks like only DDDetector,  attribute, and value are different from volumeBasedMagneticField_160812_cfi.
The ParametrizedMagneticFieldProducer above is also the same.
If this is supposed to stay for more than a week or so, it seems reasonable to refactor the common parts in volumeBasedMagneticField_160812_common_cfi
```python
 ParametrizedMagneticFieldProducer = cms.ESProducer("ParametrizedMagneticFieldProducer",
...

VolumeBasedMagneticFieldCommon = cms.PSet (
#common parts here 
)
```

Reply:
It is supposed to stay until DD4Hep becomes the default and DDD is dropped, which I guess is not happening in a week or so.
I have a slight preference for keeping the minimum number of cfi files since they quickly get confusing (one more is to be added for the DD4Hep producer from DB), but OK if you insist...

BTW could you suggest what I should do with the file that I removed (FromGun2DigiAnal_.py) which is also being addressed in another PR (see above)? Thanks.